Engine/Component/ComponentCamera.h

```diff
@@ -162,7 +163,10 @@ class ComponentCamera : public Component
 	float3 start_focus_position = float3::zero;
 	float3 goal_focus_position = float3::zero;
 
-	ClearMode camera_clear_mode = ClearMode::COLOR;
+	ClearMode camera_clear_mode = ClearMode::SKYBOX;
+
+	uint32_t skybox_uuid = 0;
+	std::shared_ptr<Skybox> camera_skybox = nullptr;
```

Comment:
Maybe having each camera in the scene an Skybox doesn't make sense, because all the cameras are in the same world and they should have the same skybox each, so why don't put the skybox logic in the Module Camera?

Reply:
I agree, skybox should be a generic thing. But also having the capability to override the skybox should be ok. So, I'm putting them in both `ModuleCamera` and `ComponentCamera`